crds/templates/namespaces.yaml

```diff
@@ -0,0 +1,64 @@
+# To prevent accidental injection into istio control plane namespaces.
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: {{ .Release.Namespace }}
+  labels:
+    istio-operator-managed: Reconcile
```

Comment:
Unrelated  to this PR: but the pattern should be to specify which operator is managing this namespace ( if that's what we want ). 

"Reconcile" is quite confusing.

Reply:
I've created https://github.com/istio/istio/issues/15825 to track this.